libraries/Microsoft.Bot.Builder.Dialogs/Constants/DialogPath.cs

```diff
@@ -1,6 +1,37 @@
-﻿namespace Microsoft.Bot.Builder.Dialogs
+﻿using System.Data;
+
+namespace Microsoft.Bot.Builder.Dialogs
 {
     public class DialogPath
     {
+        /// <summary>
+        /// Counter of emitted events.
+        /// </summary>
+        public const string EventCounter = "dialog.eventCounter";
```

Comment:
The case of these constants doesn't match the case of TurnPath constants. We should make them consistant.